templates/fast-api/README.md

```diff
@@ -26,6 +26,7 @@ The Makefile exposes the following commands:
 | `make test`           | Runs the unit test suite and test coverage  |
 | `make format`         | Runs formatting with black                  |
 | `make lint`           | Runs linting with ruff                      |
+| `make check-domain`   | Runs domain config validation
```

Comment:
Maybe add some instructions in the README to update `constants.py`?

Or alternatively you can [prompt folks for it](https://typespec.io/docs/extending-typespec/writing-scaffolding-template/#requesting-additional-user-input) in the `template.json` and then using an [interpolated value](https://typespec.io/docs/extending-typespec/writing-scaffolding-template/#interpolating-values) to set the constant with their input.